src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,102 @@
  */
 
 public class Plane {
+    private final String name;
+    private final int seats;
+    private final int wheels;
+    private final int wings;
+    private final int engines;
+    private final int windows;
 
+    public Plane(PlaneBuilder planeBuilder) {
+        this.name = planeBuilder.name;
+        this.seats = planeBuilder.seats;
+        this.wheels = planeBuilder.wheels;
+        this.wings = planeBuilder.wings;
+        this.engines = planeBuilder.engines;
+        this.windows = planeBuilder.windows;
```

Comment:
there is no need to use `this` keyword here

Reply:
Removed.